### PR TITLE
DBZ-7245 Add documentation for File Signal Channel

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-7245

### CODE REVIEW

**Comment by mfvitale** [comment]
LGTM, @ani-sha . 

@roldanbob can have a look? Thanks.

**Review by mfvitale — APPROVED** [review]

**Review by roldanbob — CHANGES_REQUESTED** [review]
@ani-sha @mfvitale I've made a couple of suggestions and also raised a couple of questions.

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 1]
```diff
@@ -291,6 +291,59 @@ The following image shows an example of how to use JConsole to send a signal:
 
 image::jmx-signal-operation.png[Using JConsole to send an `execute-snapshot` signal]
 
+== Enabling a File Signal Channel
```
[the remark this answers is not in the document]
Comment (ani-sha) [reply documentation/modules/ROOT/pages/configuration/signalling.adoc:294]: That's a question for @jpechane. Do we have any plans?

[the remark this answers is not in the document]
Comment (jpechane) [reply documentation/modules/ROOT/pages/configuration/signalling.adoc:294]: No, definitely not in the short term.

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 2]
```diff
@@ -291,6 +291,59 @@ The following image shows an example of how to use JConsole to send a signal:
...
+
+You can enable the File signaling channel by adding `file` to the `signal.enabled.channels` property in the connector configuration.
+After you enable the signaling channel, you must configure the connector to read signals from a file.
+The default file name is `file-signals.txt`.
+If you want to use a different file, you can specify the file name by setting the `signal.file` property in the connector configuration.
```
[the remark this answers is not in the document]
Comment (Naros) [reply documentation/modules/ROOT/pages/configuration/signalling.adoc:299]: Hi @roldanbob, the default location (path) is the root of the connector's classpath. If the user supplies the `signal.file` property, the property can refer to any accessible path & file available to the environment, which you would generally do in almost all situations, I would imagine.

**Comment by ani-sha** [comment]
Thanks @roldanbob! Added the changes.

**Comment by jpechane** [comment]
@ani-sha Applied, thanks

**Pull request merged**